nixos/modules/installer/cd-dvd/installation-cd-graphical-base.nix

```diff
@@ -35,8 +35,15 @@ with lib;
   # Enable sound in graphical iso's.
   hardware.pulseaudio.enable = true;
 
-  # Spice guest additions
+  # VM guest additions to improve host-guest interaction
   services.spice-vdagentd.enable = true;
+  services.qemuGuest.enable = true;
+  virtualisation.vmware.guest.enable = true;
+  virtualisation.hypervGuest.enable = true;
+  services.xe-guest-utilities.enable = true;
+  # The VirtualBox guest additions rely on an out-of-tree kernel module
+  # which lags behind kernel releases, potentially causing broken builds.
+  virtualisation.virtualbox.guest.enable = false;
```

Comment:
should be a mkDefault so users can still enable it in their own isos

Reply:
then it should be commented

Reply:
I had the same idea when I first made them mkDefault, but on second thought, having to mkForce it makes semantically more sense. If the user-created configuration wishes to authoritatively disable the feature, then the user should use mkForce anyway.

Reply:
Maybe it should be commented for the whole file why it's done... or changed for the whole file.  But I personally don't consider that a matter of this change really.